man/demi_getpeername.md

```diff
@@ -0,0 +1,47 @@
+# `demi_getpeername()`
+
+## Name
+
+`demi_getpeername` - Gets address of peer connected to this socket.
+
+## Synopsis
+
+```c
+#include <demi/libos.h>
```

Comment:
Please add `#include <sys/socket.h> /* For struct sockaddr and socklen_t. */` to indicate that this header is also required to be imported.